src/components/Root/index.tsx

```diff
@@ -15,8 +15,26 @@ import './index.module.scss'
 import './OnboardCustom.module.scss'
 import './KeystoneCustom.module.scss'
 
+type StyledThemeType = typeof styledTheme
+
+const ourStyledTheme: StyledThemeType = {
```

Comment:
turns out safe-react uses both material ui components and styled components. so there are two theme providers